doc/src/base/base.md

```diff
@@ -159,6 +159,7 @@ Base.isdispatchtuple
 
 ```@docs
 Base.isimmutable
+Base.ismutable
```

Comment:
Maybe switch the order and put `ismutable` first?

Reply:
order switched in aba863f